pkg/workload/ycsb/ycsb.go

```diff
@@ -717,7 +717,7 @@ func keyNameFromOrder(keynum uint64, zeroPadding int) string {
 // close together.
 // See YCSB paper section 5.3 for a complete description of how keys are chosen.
 func (yw *ycsbWorker) nextReadKey() string {
-	rowCount := yw.rowCounter.Last()
+	rowCount := yw.rowCounter.Last() - uint64(yw.config.insertStart)
```

Comment:
Do you mind double checking that this is how the official YCSB's `insertstart` flag behaves?